views/layouts/profiles.njk

```diff
@@ -13,18 +12,30 @@
     ) }}
 
     <main class="nudge-up">
-        <div class="page-section inner--wide-only accent--{{ pageAccent }} a--border-top">
-            <div class="page-section__content accent--{{ pageAccent }} a--border-top">
+        <div class="page-section u-inner-wide-only u-accent-border-top-{{ pageAccent }}">
+            <div class="page-section__content u-accent-border-top-{{ pageAccent }}">
                 {% for profile in profiles %}
-                    <section id="profile-{{ profile.title | slugify }}"
-                        class="content-box {% if loop.index > 1 %} accent--{{ pageAccent }} a--border-top{% endif %}">
-                        {{ profileSummary(profile) }}
-                    </section>
+                    <article id="profile-{{ profile.title | slugify }}"
```

Comment:
This was a component but it's just text content. There were no styles associated with it,  and it's only used here, so inlining this.

Reply:
👍